browser/perf/sources.gni

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/components/speedreader/common/buildflags/buildflags.gni")
+
+brave_browser_perf_sources = [
```

Comment:
Can we do a `BUILD.gn` and make it as a dependency in `//brave/browser/profiles`, instead of using a `sources.gni`?

Reply:
Thanks for the explanation, this makes sense